test/ui/transaction_list_ui_tests.js

```diff
@@ -422,4 +422,88 @@ describe('The transaction list view', function () {
     body.should.not.containSelector('#transactions-list .brand')
     body.should.not.containSelector('#brand-header')
   })
+
+  it('should render all transactions with corporate surcharge', function () {
```

Comment:
If you decide to add Cypress, I think we can keep this test as well, more like "unit" level test, nice that you added it 👍